cypress/tests/resource-provider-card.spec.ts

```diff
@@ -11,32 +11,25 @@ import { MCGMSCommon } from '../views/mcg-ms-common';
 describe('admin dashboard - resource provider card', () => {
   before(() => {
     cy.login();
-  });
-
-  after(() => {
-    cy.exec(
-      `oc delete namespacestores ${DATA_SOURCE_NAME_NSFS} -n ${DATA_FEDERATION_NAMESPACE}`,
-      { failOnNonZeroExit: false }
-    ).then(() => {
+    cy.exec(`oc delete --all namespacestores -n ${DATA_FEDERATION_NAMESPACE}`, {
+      failOnNonZeroExit: false,
+    }).then(() => {
```

Comment:
after this test finishes and before next test starts... how will be clean resources in between that ?

Reply:
the idea is that before each test, we will have a new env related to that particular test.
even though we are going to refactor, that time we will make uniform

Reply:
I am not sure if I get it...
till the time we are not refactoring, we can't have incomplete cases... we can't break the current flow just because we will be fixing it later... right now the flow we are following is every resources created in previous test should be cleaned before next test starts... once we do refactor then this flow won't be there, but till then we can't break current flow...

Reply:
I am not saying to have it incomplete. it's not incomplete also.
I mean, we are going to follow all tests will clear all the data they need before the execution of the tests.
And keeping the resources created during the test execution hanging.